gulpfile.js

```diff
@@ -68,7 +68,7 @@ function gulp_test() {
     return gulp.src('test/**/*.spec.js')
         .pipe(mocha({
             log: true,
-            timeout: 5000,
+            timeout: 30000,
```

Comment:
You can see it in my failed build: `✓ Start 2 jobs at once, should return both jobs (8150ms)` would have failed without this change.